backend/src/backend/main.py

```diff
@@ -51,4 +53,28 @@ async def test(timestamp: int):
     return {"requested_on": requested_on, "received_on": received_on}
 
 
+@api.post("/books/add/", status_code=201)
+async def books_add(book: SchemaBook):
+    await Book.objects.create(
+        id=book.id,
+        counter=book.counter,
+        period=book.period,
+        article_count=book.article_count,
+        media_count=book.media_count,
+        size=book.size,
+        url=book.url,
+        zimcheck=book.zimcheck,
+    )
+
+    for key, value in book.metadata.items():
+        await BookMetadata.objects.create(
+            book=book.id,
```

Comment:
I find reusing the above-created `Book` instance as param for the foreign-key to be a good practice. Ensures previous steps is all-good (easy to miss an `await` and end-up with unexpected state)